mne_bids/write.py

```diff
@@ -1118,7 +1118,7 @@ def write_raw_bids(raw, bids_basename, bids_root, events_data=None,
             if verbose:
                 warn('Converting to FIF for anonymization')
             bids_fname.suffix = bids_fname.suffix.replace(ext, '.fif')
-        elif kind in ['eeg', 'ieeg']:
+        elif kind in ['eeg', 'ieeg'] and ext != '.vhdr':
```

Comment:
can we avoid this maybe by reordering branching?

Reply:
I think this has to stay. It also mirrors the `ext != '.fif'` right above